crates/sui-core/src/signature_verifier.rs

```diff
@@ -212,8 +230,9 @@ impl SignatureVerifier {
 
         // Verify only the user sigs of certificates that were not cached already, since whenever we
         // insert a certificate into the cache, it is already verified.
+        // Aliases are only allowed via MFP, so CertifiedTransaction must have no aliases.
         for cert in &certs {
-            self.verify_tx(cert.data())?;
+            self.verify_tx_require_no_aliases(cert.data())?;
```

Comment:
Same question, its not safe to even have a code-path that can bypass aliases right?